src/routes/recovery/recovery.controller.ts

```diff
@@ -36,16 +34,15 @@ export class RecoveryController {
     });
   }
 
-  @HttpCode(200)
-  @Delete()
+  @HttpCode(204)
+  @Delete('/:moduleAddress')
```

Comment:
OOC: I don't think you need the slash or do you? 🤔

Reply:
It does not seem to have an influence when there or not. I'll merge this as is.